static/src/javascripts/projects/common/modules/atoms/quiz.js

```diff
@@ -0,0 +1,38 @@
+define([
+    'bean',
+    'common/utils/$',
+    'common/utils/fastdom-promise'
+], function (
+    bean,
+    $,
+    fastdom
+) {
+    return {
+        handleCompletion: function () {
+            if (HTMLFormElement.prototype.checkValidity) {
+                var $quiz = $('.js-atom-quiz--instant-reveal');
+                if ($quiz.length > 0) {
+                    bean.on(document, 'click', Array.prototype.slice.call($quiz), function (e) {
+                        var quiz = e.currentTarget;
+                        if (quiz.checkValidity()) {
+                            var $bucket__message = null,
+                                bucket = $(':checked + .atom-quiz__answer__item--is-correct', quiz).length;
+                            do {
+                                $bucket__message = $('.js-atom-quiz__bucket-message--' + bucket, quiz);
+                                if ($bucket__message.length > 0) {
+                                    fastdom.write(function () {
+                                        $bucket__message.css({
+                                            'display': 'block'
+                                        });
+                                    });
+                                    break;
+                                }
+                                bucket--;
+                            } while (bucket >= 0);
```

Comment:
ah ok no i don't think so – the `$(':checked + .atom-quiz__answer__item--is-correct', quiz)` lookup gets you the total correct answers. 

the loop, currently, will then look for the first bucket element (`'.js-atom-quiz__bucket-message--' + bucket`) with a smaller or equal total to yours (so 5/10 goes in bucket 4 of `['bucket--4', 'bucket--6']` for example